plugins/hydra_nevergrad_sweeper/tests/test_nevergrad_sweeper_plugin.py

```diff
@@ -5,43 +5,71 @@
 from typing import Any
 
 import nevergrad as ng
-import pytest
+import pytest  # type: ignore
+from hydra.core.override_parser.overrides_parser import OverridesParser
 from hydra.core.plugins import Plugins
 from hydra.plugins.sweeper import Sweeper
 from hydra.test_utils.test_utils import TSweepRunner, chdir_plugin_root
 from omegaconf import DictConfig, OmegaConf
 
 from hydra_plugins.hydra_nevergrad_sweeper import core
+from hydra_plugins.hydra_nevergrad_sweeper.nevergrad_sweeper import (
+    NevergradSweeper,  # type: ignore
+)
 
 chdir_plugin_root()
 
 
 def test_discovery() -> None:
-    assert core.NevergradSweeper.__name__ in [
+    assert NevergradSweeper.__name__ in [
         x.__name__ for x in Plugins.instance().discover(Sweeper)
     ]
 
 
 @pytest.mark.parametrize(  # type: ignore
-    "description,param_cls,value_cls",
+    "params,param_cls,value_cls",
+    [
+        ([1, 2, 3], ng.p.Choice, int),
```

Comment:
Instead of describing the expected value as the class and the type, can you just use an object?
```
([1, 2, 3], ng.p.Choice, int),
=>
 ([1, 2, 3], ng.p.Choice(values=[1,2,3], type=int) # or something like it.
```

Reply:
sounds good!